src/client/util.js

```diff
@@ -21,6 +21,22 @@ class Util {
     return code;
   }
 
+  static getWeaponDelay(weapon) {
+    var delay = 0;
+    switch(weapon) {
+      case 1:
+        delay = 1000;
+        break;
+      case 2:
+        delay = 1500;
+        break;
+      case 3:
```

Comment:
Might be nice with a comment on what weapon these numbers correspond to. Or a constant... :)